### PR TITLE
Adding the model of Bora et al. (2019)

### PR DESCRIPTION
**Pull request by mmpagani**

### CODE REVIEW

File: openquake/hazardlib/const.py [hunk 1]
```diff
@@ -105,5 +105,9 @@ class StdDev(object):
     ALL = 'All'
     #: Compute all the standard deviations for which the GMPE is defined
 
+    SINGLE_STATION = 'Single station'
+    #: Standard deviation representing ground shaking variability
+    #: within a single site.
+
```
Comment (micheles) [inline openquake/hazardlib/const.py:111]: It looks like the constant SINGLE_STATION is never used.

**Comment by micheles** [comment]
There are two files ending ing `_SAVE.csv` but I do not see how they are used. Perhaps they were committed by accident?

**Comment by mmpagani** [comment]
> There are two files ending ing `_SAVE.csv` but I do not see how they are used. Perhaps they were committed by accident?

I do not have them locally ... ??

File: openquake/hazardlib/const.py (hunk 1 above)
Comment (mmpagani) [reply openquake/hazardlib/const.py]: Fixed

**Review by micheles — APPROVED** [review]

**Comment by micheles** [comment]
Please remember to update the changelog

**Pull request merged**